docs/transaction.rst

```diff
@@ -0,0 +1,225 @@
+Transaction Operations
+======================
+
+Transact operations are similar to Batch operations, with the key differences being that the writes support the
+inclusion of condition checks, and they all must fail or succeed together.
+
+
+Transaction operations are supported using context managers. Keep in mind that DynamoDB imposes limits on the number of
+items that a single transaction can contain. PynamoDB doesn't currently handle automatic grouping or paginating, so this
```

Comment:
The note about grouping/pagination is probably unnecessary, since it would remove transaction semantics anyways